### PR TITLE
tests: Add docker network test

### LINKED ISSUE #327: docker network test
add docker integration network test

### PR DESCRIPTION
**Pull request by GabyCT**
Add integration docker network test.

Fixes #327

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: integration/docker/network_test.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	var (
+		args []string
+	)
+
+	AfterEach(func() {
+		args = []string{"network", "rm", "my-bridge-network"}
+		runDockerCommand(0, args...)
+		args = []string{"network", "ls"}
+		stdout := runDockerCommand(0, args...)
+		Expect(stdout).NotTo(ContainSubstring("my-bridge-network"))
```
Comment (jodh-intel) [inline integration/docker/network_test.go]: Nit: there are 4 occurrences of this string, so I'd be tempted to use a variable to make maintenance easier.

**Comment by GabyCT** [comment]
@jodh-intel changes applied

File: integration/docker/network_test.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("network", func() {
+	var (
+		args        []string
+		networkName string
```
Comment (jodh-intel) [inline integration/docker/network_test.go]: nit: Can't the name be set here rather than in the `BeforeEach()`?

**Comment by GabyCT** [comment]
@jodh-intel ohh ok changes applied

**Comment by devimc** [comment]
lgtm

**Pull request merged**